script/create-sitemaps.js

```diff
@@ -0,0 +1,37 @@
+const metalsmithSitemap = require('metalsmith-sitemap');
+const sitemap = require('sitemap');
+
+function createSitemaps(BUILD_OPTIONS) {
+  const hostname =
+    BUILD_OPTIONS.host === 'localhost'
+      ? 'http://localhost'
+      : BUILD_OPTIONS.host;
+
+  const metalsmithSitemapOptions = {
+    hostname,
+    omitIndex: true,
+  };
+
+  return (files, metalsmith, done) => {
+    if (BUILD_OPTIONS['brand-consolidation-enabled']) {
+      // move dynamic sitemap from sitemap.xml to sitemap-dynamic.xml
+      metalsmithSitemapOptions.output = 'sitemap-dynamic.xml';
```

Comment:
agnostic to name used- open to suggestions here. this is the name for `sitemap.xml` generated for `vets.gov`.